README.md

```diff
@@ -1,39 +1,125 @@
 
-# What's in This Repo?
+Evaluation of applications for the open grants will occur in quarterly rounds. Decisions will normally be made and communicated to applicants within eight weeks of these submission deadlines. Evaluation for RFPs takes place according to the timeline in the call. RFP-X applications are evaluated year-round but usually have longer decision times.
 
-This repo contains Requests For Proposals (RFPs) for the Grant program, which is elaborated below.
+ If you have any questions, please email us at **research@protocol.ai**.
 
- - Subscribe to this repo if you want to be updated about potential funding opportunities
+### Open Research Grants
 
-## RFPs
-Requests for Proposals are a fairly common method for funding individuals and groups to solve problems.  In our case, we may have a list of self-contained problems that likely require a sufficiently large time commitment that funding will be often provided at intervals or checkpoints, rather than upon completion.  These would have the goal of funding researchers to develop or find novel results on these topics.  RFPs are posted as issues in this repo, and a summary of active RFPs may be found below:
+Applications for open grants can be submitted at any time and are processed in quarterly blocks. The cut-offs for each quarter are:
+  * 15 February
+  * 15 May
+  * 15 August
+  * 15 November
 
-### ACTIVE:
-  - **[RFP-011: Changing the Internet](RFPs/rfp-011-changing-the-internet.md)**
-  - **[RFP-012: Checkpointing Filecoin into Bitcoin](RFPs/rfp-012-checkpointing-fil-into-btc.md)**  
+#### Protocol Labs PhD Fellowship
 
-### INACTIVE:
-  - **[RFP-001: Novel constructions for Proof-of-Replication](RFPs/rfp-001-proof-of-replication.md)**
-  - **[RFP-002: Novel constructions for Proof-of-Spacetime](RFPs/rfp-002-proof-of-spacetime.md)**
-  - **[RFP-003: Single Secret Leader Election](RFPs/rfp-003-single-leader-election.md)**
-  - **[RFP-004: Decentralised Access Control in CRDTs](RFPs/rfp-004-CRDT-ACL.md)**
-  - **[RFP-005: Optimize storage and convergence time in causal CmRDTs](RFPs/rfp-005-optimized-CmRDT.md)**
-  - **[RFP-006: Secret Single-Leader Election (SSLE)](RFPs/rfp-006-SSLE.md)**
-  - **[RFP-007: Multi-Level DHT Design and Evaluation](RFPs/rfp-007-MLDHT.md)**
-  - **[RFP-008: Scalability Bounds of P2P Pub/Sub (libp2p floodsub, gossipsub and episub)](RFPs/rfp-008-pubsub.md)**
-  - **[RFP-009: Proof of Space and Useful Space](RFPs/rfp-009-proof-of-space-and-useful-space.md)**
-  - **[RFP-010: Vector Commitments](RFPs/rfp-010-vector-commitments.md)**
+The PhD Fellowship is intended to support the development of an early-stage researcher conducting research in one of PL's fields of interest. This award is available to Ph.D. candidates who have completed their MSc degree or have otherwise satisfied the requirements for PhD candidacy.
```

Comment:
```suggestion
The PhD Fellowship is intended to support the development of an early-stage researcher conducting research in one of PL's fields of interest. This award is available to Ph.D. candidates who have completed their Master's degree or have otherwise satisfied the requirements for PhD candidacy.
```